app/javascript/react/pages/CalendarPage/CalendarPage.tsx

```diff
@@ -15,18 +23,36 @@ const CalendarPage = () => {
   const streamIdQuery = searchParams.get(STREAM_ID_QUERY_PARAMETER_NAME);
   const streamId = streamIdQuery && Number(streamIdQuery);
 
+  const fixedStreamData = useSelector(selectFixedData);
+  const movingCalendarData = useSelector(movingData);
+
   const dispatch = useAppDispatch();
 
   useEffect(() => {
-    streamId && dispatch(fetchFixedStreamById(streamId));
+    // streamId && dispatch(fetchFixedStreamById(streamId));
+    dispatch(fetchFixedStreamById(1));
   }, []);
 
+  useEffect(() => {
+    if (fixedStreamData && fixedStreamData.streamDailyAverages.length > 0) {
```

Comment:
we could also change to this shorter condition:
`fixedStreamData?.streamDailyAverages?.length`